modin/pandas/test/test_io.py

```diff
@@ -1281,6 +1281,26 @@ def test_read_parquet(self, make_parquet_file, columns):
             columns=columns,
         )
 
+    @pytest.mark.xfail(
+        condition="config.getoption('--simulate-cloud').lower() != 'off'",
+        reason="The reason of tests fail in `cloud` mode is unknown for now - issue #3264",
+    )
+    def test_read_parquet_indexing_by_column(self, make_parquet_file):
+        # Test indexing into a column of modin with various parquet file row lengths.
+        # Specifically, tests for https://github.com/modin-project/modin/issues/3527
+        # which fails when min_partition_size < nrows < min_partition_size * (num_partitions - 1)
+        from modin.config.envvars import MinPartitionSize
+
+        nrows = (
+            MinPartitionSize.get() + 1
```

Comment:
this formatting looks weird, but is apparently what happens when the line is too long (due to the comment about minimal `nrows`)